### PR TITLE
Fix docker compose

### PR DESCRIPTION
**Pull request by pjlampil**
- Fix docker compose for development
- Fix static for production
- Add initializing commands to the persister docker entrypoint
- Add logging to entrypoint
- Clean stuff up

### CODE REVIEW

File: .github/workflows/review.yml [hunk 1]
```diff
@@ -139,6 +141,16 @@ jobs:
...
+      - name: Run migrations setting
+        if: matrix.APPLY_MIGRATIONS == 1
+        run: |
+          echo "K8S_SECRET_APPLY_MIGRATIONS=1" >> $GITHUB_ENV
+
+      - name: Initialize data setting
+        if: matrix.INITIALIZE_DATA == 1
+        run: |
+          echo "K8S_SECRET_INITIALIZE_DATA=1" >> $GITHUB_ENV
+
```
Comment (Hi-Fi) [inline .github/workflows/review.yml:145]: Maybe all three above as "Persister settings" (like API setting below)?

**Review by Hi-Fi — CHANGES_REQUESTED** [review]

File: .github/workflows/review.yml (hunk 1 above)
Comment (pjlampil) [reply .github/workflows/review.yml:145]: How about now?

Comment (Hi-Fi) [reply .github/workflows/review.yml:145]: I was thinking more of one of where those 3 settings are under.

Comment (pjlampil) [reply .github/workflows/review.yml:145]: Maybe like this?

**Review by Hi-Fi — APPROVED** [review]

File: .github/workflows/review.yml (hunk 1 above)
Comment (Hi-Fi) [reply .github/workflows/review.yml:145]: Now it's clearer. OFC some if's might be redundant (and just setting the variable would be enough), but now it's at least clear to see that there isn't many common variables.

**Pull request merged**